src/amo/sagas/addonsByAuthors.js

```diff
@@ -18,13 +19,14 @@ export function* fetchAddonsByAuthors({ payload }) {
   yield put(errorHandler.createClearingAction());
 
   try {
+    const addonTypeFilter = getAddonTypeFilter(addonType);
```

Comment:
We might not need this variable, we can inline the call to `getAddonTypeFilter()` in the `filters` object below.